### DISCUSSION TITLE
Is there any way for `ge` or `conint` to show in auto completion?

### DISCUSSION THREAD

**Discussion by pedrolabonia** [pydantic/pydantic#4026] [Question] [answered]
Hey everyone!

I'm wrapping an existing API which has some constraints in the values. I wanted if possible to have autocompletion for the `ge` int types or the `conint` types.

`conint` shows up as Unknown, and using the `Field` constraints shows up as int, but it doesn't show errors on VSCode.

![image](https://user-images.githubusercontent.com/4316044/165847035-0eb9d564-8110-409d-8525-d031d1cc8f97.png)

With strict mode on, no errors are catched:
![image](https://user-images.githubusercontent.com/4316044/165846927-dd7fd934-7edf-4736-ac7a-2dbb0d703652.png)

This is in Python 3.6.8, Windows 10. The python version is limited by the API itself, which is a C dynamically loaded module.

I wonder if this is just a limitation on VSCode itself, or there's just no way for the language server to parse int constraints. Maybe I should just type out Literals, but the problem is I have values that range from 0 to 10000 for example..

**Comment by samuelcolvin** [comment, marked as answer]
I'm afraid this is a limitation of vscode.

With the plug-in, I believe you could get warnings in pycharm.

**Reply by pedrolabonia** [reply]
I figured. Thanks!
